### PR TITLE
doc: add ReleaseHOWTO document

### PR DESCRIPTION
**Pull request by Vicente-Cheng**

### CODE REVIEW

**Comment by Vicente-Cheng** [comment]
Hi @bk201, @guangbochen, 
I add the document for release howto. Please help to review it.

Thanks!

File: ReleaseHOWTO.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# How to generate a new release to harvester
+
+1. Tag the version on [harvester/node-disk-manager](https://github.com/harvester/node-disk-manager)
+ repository.
+
+    (Use the github release action. Draft a new release with the related version and its commit.)
+
+2. Bump the new version on [harvester/charts](https://github.com/harvester/charts)
+
+    - Working on the the `release` branch
```
Comment (bk201) [inline ReleaseHOWTO.md:10]: If there are chart changes, we should work in the master first and backport to release.

File: ReleaseHOWTO.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - Now, the `version` and `apiVersion` are synced on NDM. That means no matter if you want to change the content of chart or update image you need to bump these two versions.
+
+    (Check the [here](https://github.com/harvester/charts/tree/master/charts/harvester-node-disk-manager) would help to know more detail.)
+
+3. Final, bump the version on [harvester/harvester](https://github.com/harvester/charts)
```
Comment (bk201) [inline ReleaseHOWTO.md:15]:
```suggestion
3. Bump the version on [harvester/harvester](https://github.com/harvester/charts)
```

File: ReleaseHOWTO.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    - Working on [deploy/charts/harvester](https://github.com/harvester/harvester/tree/master/deploy/charts/harvester)
+    - The node-disk-manager is the dependency chart of harvester. You need to bump the related version (as above) and update the chart. (TIPS Command: `helm dependency`)
```
Comment (bk201) [inline ReleaseHOWTO.md:18]:
```suggestion
    - The node-disk-manager is a Harvester dependency chart. You need to bump the related version (as above) and update the chart. (Command: `helm dependency`)
```

**Review by bk201 — COMMENTED** [review]
Please create a doc directory and move this into it, thanks!

File: ReleaseHOWTO.md (hunk 1 above)
Comment (Vicente-Cheng) [reply ReleaseHOWTO.md]: done, please check it again.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]
LGTM thanks!

**Pull request merged**